py3status/module.py

```diff
@@ -80,6 +82,17 @@ def clear_cache(self):
             if self.config['debug']:
                 syslog(LOG_INFO, 'clearing cache for method {}'.format(meth))
 
+    def set_updated(self):
+        """
+        Mark the module as updated
+        """
+        self.py3_wrapper.notify_update(self.module_full_name)
+
+    def get_latest(self):
+        for method in self.methods.values():
+            output = method['last_output']
+        return output
```

Comment:
I've only seen this in wwan_status which I've never used.  but we should keep working as we did